Project.toml

```
@@ -40,7 +40,7 @@ LazyArtifacts = "1.6"
 Markdown = "1.6"
 Nemo = "0.51.1"
 Pkg = "1.6"
-Polymake = "0.13.1"
+Polymake = "0.13.0"
```

```suggestion
Polymake = "0.13.1"
```
This version is there for a reason, and should never be lowered

---

@varuntrehan7 this still needs to be addressed (e.g. by just using the "Commit suggestion" button here in the webinterface, or by you making the change locally and pushing it)